src/includes/getting-started-config/javascript.ember.mdx

```diff
@@ -0,0 +1,110 @@
+
+This snippet includes automatic instrumentation to [monitor the performance](/platforms/javascript/performance/) of your application, which registers and configures the Tracing integration, including custom Ember instrumentation.
+
+**Note:** To reduce the volume of performance data captured, or disable it entirely, change `tracesSampleRate` to a value between 0 and 1 in production.
+
+```javascript
+import Application from '@ember/application';
+import Resolver from 'ember-resolver';
+import loadInitializers from 'ember-load-initializers';
+import config from './config/environment';
+
+import { InitSentryForEmber } from '@sentry/ember';
+
+InitSentryForEmber();
+
+export default class App extends Application {
+  modulePrefix = config.modulePrefix;
+  podModulePrefix = config.podModulePrefix;
+  Resolver = Resolver;
+}
+```
+
+Then add the following config to your `config/environment.js`:
+
+```javascript
+ENV['@sentry/ember'] = {
+  sentry: {
+    dsn: '___PUBLIC_DSN___',
+    tracesSampleRate: 1.0, // We recommend adjusting this in production
+  }
+};
+```
+
+Once this is done, all unhandled exceptions are automatically captured by Sentry.
+
+**Important:** Note your DSN. The _DSN_ (Data Source Name) tells the SDK where to send events. If you forget it, view _Settings -> Projects -> Client Keys (DSN)_ in the Sentry web UI.
+
+
+### Ember Specifics
+
+The `@sentry/ember` addon comes with some options to manage the Ember specific instrumentation, these options are set on the addon config directly. All Sentry SDK options that would be passed to `init` should be set in the `sentry` key inside your addon config instead.
+
+```javascript
+ENV['@sentry/ember'] = {
+  // Ember specific options
+  sentry: {
+    // Sentry options
+  }
+};
+```
+
+### Ember Performance
+
+The Sentry tracing integration is already setup via the Ember addon with custom Ember instrumentation for routing, components and the runloop. It sideloads `@sentry/tracing` as a chunk to instrument your application. If you would like to disable this automatic instrumentation and no longer receive the associated transactions you can set `disablePerformance` in your config.
+
+```javascript
+ENV['@sentry/ember'] = {
+  disablePerformance: true
+};
+```
+
+Performance data is transmitted using a new event type called “transactions”, which you can learn about in [Distributed Tracing](/product/performance/distributed-tracing/). To capture transactions, you must configure your SDK to set the `tracesSampleRate` configuration to a nonzero value. The default configuration will transmit 100% of captured transactions; lower this value in production to avoid consuming your quota too quickly.
+
+### Routes
```

Comment:
How critical is this for Getting Started? Should it move to the Configuration>Basic Options page?

Reply:
It's not critical, we can definitely move it, can you link me an example of another framework that extends options?